### PR TITLE
Adding experimental image

### PR DESCRIPTION
**Pull request by muh-hassani**
I have added a directory, containing the Dockerfile and environment.yml for the experimental image. The .travis.yml file and deploy.sh were also modified accordingly.

### CODE REVIEW

File: pyiron-experimental/environment.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+channels:
+- conda-forge
+dependencies:
+- temmeta = 0.0.6
```
Comment (jan-janssen) [inline pyiron-experimental/environment.yml:4]:
```suggestion
- temmeta =0.0.6
- pyprismatic =1.2.1 
- match-series =1.0.0 
- pyxem =0.12.2
```

File: pyiron-experimental/environment.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+- pyprismatic = 1.2.1
```
Comment (jan-janssen) [inline pyiron-experimental/environment.yml:5]:
```suggestion
```

File: pyiron-experimental/environment.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+channels:
+- conda-forge
+dependencies:
+- temmeta = 0.0.6
+- pyprismatic = 1.2.1 
+- match-series = 1.0.0
```
Comment (jan-janssen) [inline pyiron-experimental/environment.yml:6]:
```suggestion
- match-series =1.0.0 
```

File: pyiron-experimental/environment.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+channels:
+- conda-forge
+dependencies:
+- temmeta = 0.0.6
+- pyprismatic = 1.2.1 
+- match-series = 1.0.0 
+- pyxem = 0.12.2
```
Comment (jan-janssen) [inline pyiron-experimental/environment.yml:7]:
```suggestion
- pyxem =0.12.2
```

File: pyiron-experimental/environment.yml (hunk 1 above)
Comment (muh-hassani) [reply pyiron-experimental/environment.yml:4]: The spaces were for better readability. I will remove these.

**Comment by jan-janssen** [comment]
Maybe it is good to add some example notebooks for this image as well, then we can also test the image by executing the notebooks.

**Comment by muh-hassani** [comment]
Sure, we have number of them in different repositories. I will include them after the technical meeting.

**Comment by jan-janssen** [comment]
I start to like the unit tests, they fail when we forgot something, like adding the example files or missing dependencies. We should make sure to add more unit tests.

**Review by jan-janssen — APPROVED** [review]
Looks good to me

**Pull request merged**